.gitignore

```diff
@@ -64,9 +64,6 @@ docs/antora-aissemble-ui/public/
 **/charts/*
 Chart.lock
 extensions/extensions-helm/aissemble-spark-operator-chart/values.yaml
-extensions/extensions-helm/aissemble-fastapi-chart/values.yaml
```

Comment:
I: Each of these charts has the same pattern:
1. remove the values file from gitignore
2. rename values.template.yaml to values.yaml
3. set the tag to an empty string in values.yaml
4. remove the replacer plugin execution from the POM